projects/tangent-protocol/index.js

```diff
@@ -0,0 +1,14 @@
+const { get } = require('../helper/http')
+
+module.exports = {
+  misrepresentedTokens: true,
+  cardano: {
+    tvl: async () => {
+      const data = await get('https://tangent-staking-mainnet.herokuapp.com/pool/getTANGData')
```

Comment:
sorry, we dont like relying on 3rd party api for tvl data, it is blackbox and can break in the future, I found that you store staked tokens here? https://cardanoscan.io/address/01549a01adb71a806a13ec4098006e9d8aaa450ca81db4bbd1a199fe5c700131851951169f18c6cb44863e250c39aea44cc3aef5d66a32b761

I can refactor your adapter to comput staking tvl from that, but will need to fix pricing of TANG & Lp token on our backend first.

Regarding counting NFTs as tvl, sorry, right now, we dont price ADA nfts, so cant include it

Reply:
@g1nt0ki 
Could you please give me more details about the third-party service that was mentioned? The https://tangent-staking-mainnet.herokuapp.com/pool/getTANGData is an API that we utilize to determine the value of staked tokens and NFTs by gathering data from our database and retrieving pricing data from CoinGecko.

Reply:
@g1nt0ki 
If the solution mentioned is the only option available and aligns with your policies, please proceed with refactoring and fixing the pricing on your backend. Additionally, is there anything else that we need to do in the meantime to ensure that we are listed on DefiLlama as soon as possible? Thank you for your help.

Reply:
@g1nt0ki 

I just have two more questions to confirm your suggestion:

- Can you please confirm that all our staked tokens will be stored at the address that will be used for calculating the TVL?
- Additionally, would you like to include our treasury address, which receives fees generated from the app, in the TVL calculation?

Reply:
hi @marjankhz , to us, you are the third party api. 

Yes, we will make a listing like this: https://defillama.com/protocol/genius-yield (enable the staking toggle top right corner)

if you share the treasury address, we can track it under treasury metric like https://defillama.com/protocol/parrot-protocol?treasury=true

Reply:
Hello @g1nt0ki, I understand what you're saying and it makes sense :)

Thank you for the suggestion, I will update our treasury address in the previous note and also add it here. If there is anything else we can assist you with, please let us know. Thank you once again.

Reply:
@g1nt0ki 
Hello, I have just included our treasury address in the following text:
`addr1q9d4sdume9sn8kfmnes07rzvj45l4dpmjzvukm730pg3upyxj7hwpxu5tght0zquuapxgugqpzhhg3ej9xkk9kf0arpq92g6a4`

Reply:
@g1nt0ki 
Hey, just checking in to see if there's anything else we need to take care of. Could you also let me know how long this process is expected to take and if we'll be receiving updates through this PR? Thanks!